src/cljs/rems/util.cljs

```diff
@@ -3,6 +3,16 @@
             [re-frame.core :as rf]
             [secretary.core :as secretary]))
 
+(defn andstr
+  "Like `apply str` but only produces something if all values are truthy like `and`.
+
+  (andstr nil 1 2 3) => nil
+  (andstr true 1 false 3) => nil
+  (andstr (:foo {:foo 2}) \"/\") => \"2/\""
```

Comment:
Instead of having the examples only as documentation, move them to a unit test for some executable documentation.

Reply:
Done in another PR.